system_tests/program_test.go

```diff
@@ -84,6 +84,16 @@ func keccakTest(t *testing.T, jit bool) {
 	if otherVersion != programVersion {
 		Fatal(t, "mismatched versions", stylusVersion, programVersion)
 	}
+	programSize, err := arbWasm.ProgramSize(nil, programAddress)
+	Require(t, err)
+	if programSize < 5000 || programSize > 20000 {
+		Fatal(t, "unexpected size", programSize)
+	}
+	programMemoryFootprint, err := arbWasm.ProgramMemoryFootprint(nil, programAddress)
+	Require(t, err)
+	if programMemoryFootprint != 1 {
+		Fatal(t, "unexpected memory footprint", programMemoryFootprint)
+	}
```

Comment:
Lets move this to the memory test, which has an example that's known to be 120 pages :)

Reply:
:+1: